### ISSUE TITLE
Preview with housenumber addition seems to be broken with update.

### ISSUE THREAD

**Issue by ericclaeren** [experius/Magento-2-Module-Experius-Postcode-NL#58]
Hi, It seems the formatAddress() breaks the preview functionality and setting the correct addition value.

https://github.com/experius/Magento-2-Module-Experius-Postcode-NL/compare/1.3.4...1.7.0#diff-beb6d976e0f41c9953ccbacb72f20d30419a2d8208bb335920a4bfa1dc63bc98R559

This line:
```
var houseNoAdd = overrideData.housenumber_addition || formData.experius_postcode_housenumber_addition_manual;
```

Only seems to set the addition from the manual selection. This probably should be:
```
var houseNoAdd = overrideData.housenumber_addition || formData.experius_postcode_housenumber_addition || formData.experius_postcode_housenumber_addition_manual;
```

**Comment by rubenexp** [comment]
Hi @barryvdh @ericclaeren 
Your PR will cause the manual field to be set even if its hidden, i've added my own PR with the correct fix.

**Comment by rubenexp** [comment]
Fixed in release 1.8.2, PR #60

**Comment by ericclaeren** [comment]
Great! Thanks!
